zookeeper/src/main/java/com/linecorp/armeria/client/zookeeper/ZooKeeperEndpointGroup.java

```diff
@@ -54,81 +55,145 @@
      * Returns a new {@link ZooKeeperEndpointGroup} that retrieves the {@link Endpoint} list from
      * the ZNode at the specified connection string and path. A new ZooKeeper client will be created internally.
      * The ZooKeeper client will be destroyed when the returned {@link ZooKeeperEndpointGroup} is closed.
+     *
+     * @param zkConnectionStr the ZooKeeper connection string
+     * @param zNodePath the ZooKeeper node to register
      */
     public static ZooKeeperEndpointGroup of(String zkConnectionStr, String zNodePath) {
```

Comment:
If we chose to use `legacy` instead of `default`, we're gonna have to always require a `DiscoverySpec`.